### DISCUSSION TITLE
Github Education Pack

### DISCUSSION THREAD

**Discussion by pranavbalu1** [community/community#137878] [GitHub Education]
### Select Topic Area

Question

### Body

My requests for the education plan keeps getting rejected, because my school doesn't have an ID with a "valid through" date or any kind of document that properly indicates my attendance. Any other ideas to get Student status verified?

**Comment by davevad93** [comment]
Hi @pranavbalu1 ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), "***documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter. If your student ID doesn't have a current date, you can provide some of the above documents.***"

You can also use as proof a photo of your last exam results, as long as it shows a current date in a `DD/MM/YYYY` or `MM/DD/YYYY` format, otherwise the application system won't be able to assess your current student status.

**Reply by pranavbalu1** [reply]
Ok, so I got approved about 3 days back! But I'm still not able to gain access to copilot? I'll wait another day before contacting customer support

**Reply by davevad93** [reply]
Hey @pranavbalu1, if your status label is green, you've been approved but that doesn't mean your benefits are available. When they become available the status label will turn purple and change to:

- `Congrats! Your request for @pranavbalu1 was approved and your benefits are now available.`

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

For more reference, check the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) and also the following resolved discussions:
  #132790 , #131665 , #132470 , #133531, #134279, #135849

If after receiving your benefits you still can't access Copilot, then you should definitely contact support by raising a ticket [here](https://support.github.com/contact/education), select 
"`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Comment by pg56714** [comment]
@pranavbalu1 
![image](https://github.com/user-attachments/assets/90e00493-02bf-4975-a86c-54e1a4141ea8)


Mark me as answer thanks

[copilot](https://github.com/features/copilot)

Click **Get started with Copilot**.

Click **Start a free trial**.

That's awesome! It really is free to use, with the big text saying, **Congratulations! You are eligible to use GitHub Copilot for free**.

![image](https://github.com/user-attachments/assets/3db0309c-4888-435a-9378-e57d850093c2)
![image](https://github.com/user-attachments/assets/bb49cfe9-4cde-460e-9503-a38b28fb3648)
![image](https://github.com/user-attachments/assets/d3fa07f3-d37f-4561-9d40-39c0f879fdf7)

**Comment by diveshthapa** [comment]
Does this available in country Nepal ?
![NepalFlagGIF](https://github.com/user-attachments/assets/52415b8f-ca38-465b-ab8e-ff9aed926782)

**Reply by samus-aran** [reply]
Yes, the Student Developer Pack is available to those applying from Nepal. 👍

**Reply by diveshthapa** [reply]
Mark me as answer, thank you ☺
